File: main (1).c
/******************************************************************************

Cilene - 16/02/23 - Calculadora em C

*******************************************************************************/
#include <stdio.h>

int soma (int a, int b)
{
    return a+b;
}
int subtracao (int a, int b)
{
    return a-b;
}
int prod (int a, int b)
{
    return a*b;
}
int resto (int a, int b)
{
    return b%a;
}
int divisao (int a, int b)
{
    return a/b;
}


int main()
{
    int a=22, b=223;
    
    // Saída
    printf ("Soma = %d", soma(a,b));
    printf ("\nSubtracao = %d", subtracao(a,b));
    printf ("\nProduto = %d", prod(a,b));
    printf ("\nDivisao = %d", divisao(a,b));
    printf ("\nResto = %d", resto(a,b));

    return 0;
}

File: Main (1).java
/******************************************************************************

*******************************************************************************/
public class Main
{
	public static void main(String[] args) {
	
	 int a=10, b=2;
     System.out.println("Soma = "+ soma(a,b));
     System.out.println("Subtracao = "+ subtracao(a,b));
     System.out.println("Produto= "+ prod(a,b));
     System.out.println("Resto = "+ resto(a,b));
     System.out.println("Divisao = "+ divisao(a,b));

	}
	public static int soma(int a, int b){
        return a+b;
    }
    	public static int subtracao(int a, int b){
        return a-b;
    }
    	public static int prod(int a, int b){
        return a*b;
    }
    	public static int resto(int a, int b){
        return b%a;
    }
    	public static int divisao(int a, int b){
    	    if(b==0)return 0;
        return a/b;
    }
    
}
